# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/qjackctl/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/qjackctlConnectionsForm.ui.orig	2005-10-28 13:33:05.000000000 +0200
+++ ./src/qjackctlConnectionsForm.ui	2005-10-28 13:33:11.000000000 +0200
@@ -439,7 +439,5 @@
 <pixmapinproject/>
 <layoutdefaults spacing="4" margin="4"/>
 <includehints>
-    <includehint>qjackctlconnectview.h</includehint>
-    <includehint>qjackctlconnectview.h</includehint>
 </includehints>
 </UI>
--- ./src/qjackctlPatchbayForm.ui.orig	2005-10-28 13:37:42.000000000 +0200
+++ ./src/qjackctlPatchbayForm.ui	2005-10-28 13:37:50.000000000 +0200
@@ -885,6 +885,5 @@
 <pixmapinproject/>
 <layoutdefaults spacing="4" margin="4"/>
 <includehints>
-    <includehint>qjackctlpatchbayview.h</includehint>
 </includehints>
 </UI>
